salt/pillar/varstack_pillar.py

```diff
@@ -21,27 +21,20 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-# Import python libs
-import logging
-
-HAS_VARSTACK = False
 try:
     import varstack
-    HAS_VARSTACK = True
 except ImportError:
-    pass
-
-# Set up logging
-log = logging.getLogger(__name__)
+    varstack = None
 
 # Define the module's virtual name
 __virtualname__ = 'varstack'
 
 
 def __virtual__():
-    if not HAS_VARSTACK:
-        return False
-    return __virtualname__
+    return (
+        varstack and __virtualname__ or False,
+        'The varstack module could not be loaded: varstack dependency is missing.'
+    )
```

Comment:
This looks weird, what is it going to return?  Why are we not just using.
```
if varstack is None:
    return False, 'The varstack module could not be loaded: varstack dependency is missing.'
return __virtualname__

Reply:
It is going to return `__virtualname__` if `varstack` is a module, `False` otherwise. The error message is ignored.

Two returns are bad style in Python.

Reply:
I would still contend that the other way is more readable, but whatever

Reply:
Do you want me to change this part?  I see several +1s, and actually, this is how it looked in my initial PR.

Reply:
you are fine.  just waiting for @isbm to :+1: the review.